src/Job.js

```diff
@@ -142,16 +147,14 @@ class Job extends EventEmitter {
       // Get job status.
       let req = request
```

Comment:
@plattdl Hey thanks for updating these!  We can also remove the variable assignment (let req = ) here and just call the request.*.end() function like we did before.